index.html

```diff
@@ -236,6 +236,32 @@ <h4 class="connect">LET'S CONNECT</h4>
       </div>
 
     </section>
+    <section class="form-container">
+      <div class="form-min">
+      <div class="form-des">
+        <h1>Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      </div>
+      <form action="POST">

```

Comment:
Your form should submit the data to formspree service. `action` should contain the target where you are sending data, in your case, it would be fromspree url, and the `method` attribute which describes the HTTP method to be used when submitting the form data.